src/components/NewPost/NewPost.js

```diff
@@ -1,51 +1,150 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import { Link } from 'react-router-dom';
 
 import styles from './NewPost.module.css';
 
 const NewPost = () => {
+  const [inputState, setInputState] = useState({
+    id: '',
+    name: '',
+    url: '',
+    date: '',
+    place: '',
+    tags: '',
+  });
+  const submitHandler = (event) => {
+    event.preventDefault();
+    console.log(inputState);
+  };
   return (
-    <form className={styles.Form}>
-      <Link
-        to={{
-          pathname: '/',
-        }}
-      >
-        <button className={styles.Button} type="submit">
-          Zapisz
-        </button>
-      </Link>
-      <Link
-        to={{
-          pathname: '/',
-        }}
-      >
+    <form className={styles.Form} onSubmit={submitHandler}>
+      <button className={styles.Button} type="submit">
+        Zapisz
+      </button>
+
+      <Link to="/">
         <button className={styles.Button}>Anuluj</button>
       </Link>
       <div className={styles.Option}>
-        <label className={styles.Label} htmlFor="Form-URL">
+        <label className={styles.Label} htmlFor="name">
+          Nazwa
+        </label>
+        <input
+          className={styles.Input}
+          type="text"
+          name="name"
+          id="name"
+          value={inputState.name}
+          onChange={(event) => {
+            const newValue = event.target.value;
+            setInputState({
+              id: inputState.id,
+              name: newValue,
+              url: inputState.url,
+              date: inputState.date,
+              place: inputState.place,
+              tags: inputState.tags,
+            });
```

Comment:
Zastanów się, czy dałabyś radę wyodrębnić powyższą część do osobnej funkcji w taki sposób, by móc ją wykorzystać w każdym miejscu, w którym zmieniasz dowolne pole nowej pozycji na naszej liście zdjęć. Czy to jest: 
name: newValue
czy też: 
date: newValue

Zastanów się do spotkania, co jest blokujące jeśli chodzi o zrobienie takiej uniwersalnej funkcji :)